test/tools/libtesteth/TestHelper.cpp

```diff
@@ -624,13 +625,7 @@ void requireJsonFields(json_spirit::mObject const& _o, string const& _section,
 
 string prepareVersionString()
 {
-    // cpp-1.3.0+commit.6be76b64.Linux.g++
-    string commit(DEV_QUOTED(ETH_COMMIT_HASH));
-    string version = "cpp-" + string(ETH_PROJECT_VERSION);
-    version += "+commit." + commit.substr(0, 8);
-    version +=
-        "." + string(DEV_QUOTED(ETH_BUILD_OS)) + "." + string(DEV_QUOTED(ETH_BUILD_COMPILER));
-    return version;
+    return string{"testeth "} + eth_get_buildinfo()->project_version;
```

Comment:
It was useful for Dimitry to have exact commit here, I think it writes it to the filled test files.
cc @winsvega

Reply:
Ok, let me know what exactly is needed.

Reply:
Ah `project_version` it includes commit already?
Ok then

Reply:
The comment you deleted here showed how it looks in test files.
I think commit and "cpp" are the most important parts

Reply:
Not always. Only if the git is not directly on a version tag. https://github.com/ethereum/cable/blob/master/buildinfo/buildinfo.cmake#L32-L49

For tagged releases the commit hash is omitted.

Reply:
Probably that's fine